src/start-selenium-node.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# set -e: exit asap if a command exits with a non-zero status
+set -e
+
+echo "Starting Selenium Grid Node Docker..."
+
+if [[ -z "${SE_EVENT_BUS_HOST}" ]]; then
+  echo "SE_EVENT_BUS_HOST not set, exiting!" 1>&2
+  exit 1
+fi
+
+if [[ -z "${SE_EVENT_BUS_PUBLISH_PORT}" ]]; then
+  echo "SE_EVENT_BUS_PUBLISH_PORT not set, exiting!" 1>&2
+  exit 1
+fi
+
+if [[ -z "${SE_EVENT_BUS_SUBSCRIBE_PORT}" ]]; then
+  echo "SE_EVENT_BUS_SUBSCRIBE_PORT not set, exiting!" 1>&2
+  exit 1
+fi
+
+if [ ! -z "$SE_OPTS" ]; then
+  echo "Appending Selenium options: ${SE_OPTS}"
+fi
+
+if [ ! -z "$SE_NODE_GRID_URL" ]; then
+  echo "Appending Grid url: ${SE_NODE_GRID_URL}"
+  SE_GRID_URL="--grid-url ${SE_NODE_GRID_URL}"
+fi
+
+java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar node \
+  --publish-events tcp://"${SE_EVENT_BUS_HOST}":${SE_EVENT_BUS_PUBLISH_PORT} \
+  --subscribe-events tcp://"${SE_EVENT_BUS_HOST}":${SE_EVENT_BUS_SUBSCRIBE_PORT} \
+  --detect-drivers false \
+  --config /opt/bin/config.toml \
+  ${SE_GRID_URL} ${SE_OPTS}
```

Comment:
@budtmo here is where it's required to use the `selenium-server.jar` in order to make the connection between the hub and the node